### ISSUE TITLE
PostgresSchema incorrectly handling time (without time zone)

### ISSUE THREAD

**Issue by jmjjg** [cakephp/cakephp#7363]
Using PHP 5.6.8, PostgreSQL 8.4.16 and CakePHP 3.1.*-dev ( https://github.com/cakephp/cakephp.git 45227cecf33a7a1ee16f411dc59174c1c4f7d6c6 ), but I believe all CakePHP 3.x is affected.

If I create a "time" column in PostgreSQL, CakePHP recognizes it as text.

Sorry for the presentation below (I'm new here).

Steps to reproduce:
1°) in tests/TestCase/Database/Schema/PostgresSchemaTest.php
a°) Insert at line 74
```sql
--views SMALLINT DEFAULT 0,
readingtime TIME,
--created TIMESTAMP,
```
b°) Insert at line 112
```php
/*[
    'TIME',
    ['type' => 'time', 'length' => null]
],*/
[
    'TIME WITHOUT TIME ZONE',
    ['type' => 'time', 'length' => null]
],
/*// Integer
[
    'SMALLINT',
    ['type' => 'integer', 'length' => 5]
],*/
```
c°) Insert at line 338
```php
/* 'views' => [
     'type' => 'integer',
     'null' => true,
     'default' => 0,
     'length' => 5,
     'precision' => null,
     'unsigned' => null,
     'comment' => null,
     'autoIncrement' => null,
 ],*/
 'readingtime' => [
     'type' => 'time',
     'null' => true,
     'default' => null,
     'length' => null,
     'precision' => null,
     'comment' => null,
 ],
 /*'created' => [
     'type' => 'timestamp',
     'null' => true,
     'default' => null,
     'length' => null,
     'precision' => null,
     'comment' => null,
 ],*/
```
2°) Run the tests and watch them fail with time being detected as text:
```bash
vendor/bin/phpunit --verbose --debug vendor/cakephp/cakephp/tests/TestCase/Database/Schema/PostgresSchemaTest.php
```
3°) Apply fix in src/Database/Schema/PostgresSchema.php, insert line 88
```php
/*if (strpos($col, 'timestamp') !== false) {
    return ['type' => 'timestamp', 'length' => null];
}*/
if (strpos($col, 'time') !== false) {
    return ['type' => 'time', 'length' => null];
}
/*if ($col === 'serial' || $col === 'integer') {
    return ['type' => 'integer', 'length' => 10];
}*/
```
4°) Run the tests and watch them pass. :)

**Comment by ADmad** [comment]
@jmjjg The issue/comment has a preview feature plus there is a link which shows how to use markdown to properly format your post :)

**Comment by markstory** [comment]
Fix in #7378
